src/Bridges/Constraint/Constraint.jl

```diff
@@ -81,6 +81,8 @@ Add all bridges defined in the `Bridges.Constraint` submodule to
 `bridged_model`. The coefficient type used is `T`.
 """
 function add_all_bridges(bridged_model, ::Type{T}) where {T}
+    MOIB.add_bridge(bridged_model, GreaterToIntervalBridge{T})
```

Comment:
Add `if T <: AbstractFloat`, this should fix the tests

Reply:
Done